images/mosdepth/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM debian:bookworm-slim AS base
+
+ARG VERSION=${VERSION:-0.3.11}
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+
+RUN apt-get update && apt-get install -y \
+        curl && \
+    rm -r /var/lib/apt/lists/* && \
+    rm -r /var/cache/apt/* && \
+    curl -L -o '/usr/local/bin/mosdepth' https://github.com/brentp/mosdepth/releases/download/v${VERSION}/mosdepth && \
+    chmod a+x /usr/local/bin/mosdepth
```

```suggestion
ADD https://github.com/brentp/mosdepth/releases/download/v0.3.11/mosdepth /usr/local/bin/mosdepth
RUN a+x /usr/local/bin/mosdepth
```

no strong feelings about this

---

Thanks for the suggestions. implemented.